deployments/liqo/subcharts/authService/templates/auth-service-deploy.yaml

```diff
@@ -159,19 +161,24 @@ spec:
           - "--resyncSeconds"
           - "30"
           - "--listeningPort"
+          {{ if .Values.ingress.enable }}
           - "5000"
+          {{ else }}
+          - "443"
```

Comment:
Is it necessary to use two different ports? I feel it increases the complexity (e.g. in the service definition).

Reply:
No, it's not necessary, but I think that it's better because if we expose the service with a non-TLS backend on port 443 some ingress controller may intend it as a TLS backend [(traefik docs)](https://doc.traefik.io/traefik/v1.7/configuration/backends/kubernetes/#tls-communication-between-traefik-and-backend-pods)

Vice versa, exposing a TLS service on a non 443 port can be strange and hard to understand

Note: we expose the pod port with TLS if we are exposing it with a NodePort, without TLS if we are using ingresses

Reply:
Ok, got it